segment_source_resource/log.py

```diff
@@ -63,7 +63,7 @@ def std_processor(_, method_name, event_dict):
 
 def configure(levelname, **global_values):
     try:
-        level = structlog.stdlib.INFO
```

Comment:
Using logging is nice but curious why we need this change?

Reply:
this was deprecated